README.md

```diff
@@ -0,0 +1,5 @@
+# eslint-config
+
+Need devDependencies in the package.json:
```

Comment:
If you add a package.json here you could include the devDependencies here and we can put this on npm as `@seed-media/eslint-config` and we can just include that as a devDependency. Allows us to control our linting deps from this one place then.